peer.go

```diff
@@ -116,11 +116,12 @@ func (l *PeerList) Add(hostPort string) *Peer {
 	return p
 }
 
-// Get returns a peer from the peer list, or nil if none can be found.
-func (l *PeerList) Get(prevSelected map[string]struct{}) (*Peer, error) {
+// GetNew returns a new, previously un-selected peer from the peer list, or nil,
+// if no new un-selected peer can be found.
```

Comment:
@prashantv I'd wondered that, though I wasn't sure it was really an "error" because returning `nil` for the peer in that case is the expected behavior... Then the client has to do weirder error handling to actually "deal" with that `ErrNoNewPeers`, and then should they *still* check for `nil` ?